docs/source/references/config_files/project/base_yml.rst

```diff
@@ -161,8 +161,32 @@ these settings have to be their default values as shown in the
 
       :port:
         | The port on which the service can be reached
+        | ``integer``
+
+  :communicationStore:
```

Comment:
I think this is for managing subscriptions in emails and notification. @pradumangoyal might be able to explain better.

Reply:
Yes they are used to handle the notification subscriptions and objects